openedx/core/djangoapps/theming/urls.py

```diff
@@ -16,3 +16,5 @@
             name='openedx.theming.update_theme_fragment_view',
         ),
     ]
+else:
+    urlpatterns = []
```

Comment:
My version of this change is here:
https://github.com/edx/edx-platform/pull/16775/commits/ed02841a29ff3df8f45b016b12aa5e3d8ac098d2
The race is on! 😸